corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py

```diff
@@ -706,9 +735,8 @@ def test_reassign_case_and_sync(self):
             form_extras={'last_sync_token': None}
         )
         assert_user_has_case(self, self.user, case.case_id, restore_id=self.sync_log._id)
-        payload = generate_restore_payload(self.project, self.user, restore_id=self.sync_log._id, version=V2)
-        next_sync_log = synclog_from_restore_payload(payload)
-        self.assertFalse(next_sync_log.phone_is_holding_case(case.case_id))
+        next_sync_log = self.get_next_sync_log(restore_id=self.sync_log._id, version=V2)
+        assert_user_doesnt_have_case(self, self.user, case.case_id, restore_id=next_sync_log._id)
```

Comment:
This was changed from
```py
self.assertFalse(next_sync_log.phone_is_holding_case(case.case_id))
```
The change was necessary because the `assert_user_has_case` and `get_next_sync_log` both sync against the same sync log id, so it's necessary to sync again (which is done by `assert_user_doesnt_have_case`) to check of the user doesn't have the case.